boards/arm/sam4s_xplained/ext_ram.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Justin Watson 2017
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <init.h>
+#include <nanokernel.h>
+#include <pinmux.h>
+#include <soc.h>
+#include <sys_io.h>
+
+/* Static Memory Controller (SMC) */
```

Comment:
Maybe it would be better to add support for SMC and the external IS66WV51216DALL SRAM in a separate patch. It's quite a job. One more remark: shouldn't we also update the linker script to let the compiler know we have more RAM?